src/utils/playControl.js

```diff
@@ -1,21 +1,39 @@
 import Hls from "hls.js";
 
+import { getChannelInfo } from "@/apis/radioChannels";
+
 let hlsInstance = null;
 
-const controlStreamingPlayback = (videoId, videoUrl, isPlaying) => {
-  const video = videoId.current;
+const controlStreamingPlayback = async (
+  videoRef,
+  selectedChannelId,
+  isPlaying
+) => {
+  const video = videoRef.current;
+
+  if (!isPlaying) {
+    try {
+      const { data } = await getChannelInfo(selectedChannelId);
+      const url = data.url;
```

Comment:
```suggestion
      const { data: url } = await getChannelInfo(selectedChannelId);
```
구조 분해 할당으로 한 줄로 줄일 수 있을 것 같아요!

Reply:
좋은 포인트 감사해요~
다만, `data.url`은 구조 분해 할당하려면
```
const { data: { url } } = await getChannelInfo(selectedChannelId);
```
이런 형태로 작성되어야 해서
저는 오히려 가독성이 떨어질 수 있다고 생각하는데
두 분 의견이 궁금합니다~
@devjaeyoon @Eun0713

Reply:
재윤님이 제안해주신 구조분해 방식도 좋은 방식이라고 생각해요!
그렇지만 확인해보니 구조분해 방식을 사용하면 프리티어 줄바꿈 기준을 넘겨서 자동으로 여러 줄로 나뉘어서 가독성이 떨어지는 느낌이 있었어요😂
그래서 저는 개인적으로 이번에는 기존 방식인  `data.url`로 작성하는 것이 나을 거 같다고 생각합니다‼️

Reply:
모두 의견 감사합니다!
구조 분해 방식도 충분히 좋은 접근이라고 생각해요!
이번 경우엔 줄바꿈으로 인해 오히려 가독성이 떨어질 수 있다는 점에서 `data.url`을 유지하겠습니다 😊